.gitignore

```diff
@@ -1 +1,7 @@
 .idea
+/wiz-kubernetes-connector/charts/wiz-broker-*.tgz
```

Comment:
Please check that you don't break this [script](https://github.com/wiz-sec/charts/blob/master/.circleci/package_index.sh#L41)